site/src/components/AppLink/AppLink.tsx

```diff
@@ -60,36 +65,50 @@ export const AppLink: FC<PropsWithChildren<AppLinkProps>> = ({
   ) : (
     <ComputerIcon />
   )
-  let tooltip = ""
+
+  let shareIcon = <LockOutlinedIcon />
+  let shareTooltip = "This app is private, only you can access it"
```

Comment:
Shorten to match similar verbiage as others
```suggestion
  let shareTooltip = "Private, only accessible by you"
```